### ISSUE TITLE
[brain] numpy.ndarray.T infers to None

### ISSUE THREAD

**Issue by eugene57** [PyCQA/astroid#814]
### Steps to reproduce
Lint following file:
```
import numpy

x = numpy.ndarray([1, 2])
print(x.T[0])
```

### Current behavior
```
************* Module numpy_ndarray
numpy_ndarray.py:4:6: E1136: Value 'x.T' is unsubscriptable (unsubscriptable-object)
```
This is likely caused by https://github.com/PyCQA/astroid/blob/master/astroid/brain/brain_numpy_ndarray.py#L20
Probably will be better to have `self.T = np.ndarray([0, 0])`

### Expected behavior
No errors

### ``python -c "from astroid import __pkginfo__; print(__pkginfo__.version)"`` output
2.4.1

**Comment by hippo91** [comment]
@eugene57 thanks again for this other report.
Once again you are right, but this issue is directly related to PyCQA/pylint#3387 and thus should be corrected thanks to PR #831.
Note that the correction i made is exactly the one you proprosed! If you face other problem with the ``numpy`` brains do not hesitate to submit a PR and i will be pleased to review it and help you.

**Comment by eugene57** [comment]
@hippo91 thank you!
Unfortunately I cannot directly contribute code due to some legal restrictions.
